Signal/src/contact/SystemContactsFetcher.swift

```diff
@@ -143,12 +143,12 @@ class AddressBookContactStoreAdaptee: ContactStoreAdaptee {
         assert(self.changeHandler == nil)
         self.changeHandler = changeHandler
 
-        NotificationCenter.default.addObserver(self, selector: #selector(runChangeHandler), name: kAddressBookContactyStoreDidChangeNotificationName, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(runChangeHandler), name: kAddressBookContactStoreDidChangeNotificationName, object: nil)
 
         let callback: ABExternalChangeCallback = { (_, _, _) in
             // Ideally we'd just call the changeHandler here, but because this is a C style callback in swift, 
             // we can't capture any state in the closure, so we use a notification as a trampoline
-            NotificationCenter.default.post(name: kAddressBookContactyStoreDidChangeNotificationName, object: nil)
+            NotificationCenter.default.postNotificationNameAsync(kAddressBookContactStoreDidChangeNotificationName, object: nil)
```

Comment:
This is used locally to update in response to an async event. It can be safely delivered a bit later.